src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work");
```

Comment:
The message 'Excavator started its work' could be more informative by specifying that it is an Excavator machine that has started working.